core/modules/css/testFacade.cc

```diff
@@ -99,28 +99,28 @@ struct FacadeFixture {
         kv.push_back(make_pair("/DBS/dbC", ""));
         p = "/DBS/dbA/TABLES";
         kv.push_back(make_pair(p, ""));
-        kv.push_back(make_pair(p + "/Object", ""));
+        kv.push_back(make_pair(p + "/Object", "READY"));
         kv.push_back(make_pair(p + "/Object/partitioning", ""));
         kv.push_back(make_pair(p + "/Object/partitioning/lonColName", "ra_PS"));
         kv.push_back(make_pair(p + "/Object/partitioning/latColName", "decl_PS"));
         kv.push_back(make_pair(p + "/Object/partitioning/subChunks", "1"));
         kv.push_back(make_pair(p + "/Object/partitioning/dirColName","objectId"));
-        kv.push_back(make_pair(p + "/Source", ""));
+        kv.push_back(make_pair(p + "/Source", "READY"));
         kv.push_back(make_pair(p + "/Source/partitioning", ""));
         kv.push_back(make_pair(p + "/Source/partitioning/lonColName", "ra"));
         kv.push_back(make_pair(p + "/Source/partitioning/latColName", "decl"));
         kv.push_back(make_pair(p + "/Source/partitioning/subChunks", "0"));
-        kv.push_back(make_pair(p + "/FSource", ""));
+        kv.push_back(make_pair(p + "/FSource", "READY"));
         kv.push_back(make_pair(p + "/FSource/partitioning", ""));
         kv.push_back(make_pair(p + "/FSource/partitioning/lonColName", "ra"));
         kv.push_back(make_pair(p + "/FSource/partitioning/latColName", "decl"));
         kv.push_back(make_pair(p + "/FSource/partitioning/subChunks", "0"));
-        kv.push_back(make_pair(p + "/Exposure", ""));
+        kv.push_back(make_pair(p + "/Exposure", "READY"));
 
         p = "/DBS/dbB/TABLES";
         kv.push_back(make_pair(p, ""));
-        kv.push_back(make_pair(p + "/Exposure", ""));
-        kv.push_back(make_pair(p + "/MyObject", ""));
+        kv.push_back(make_pair(p + "/Exposure", "READY"));
+        kv.push_back(make_pair(p + "/MyObject", "READY"));
         kv.push_back(make_pair(p + "/MyObject/partitioning", ""));
         kv.push_back(make_pair(p + "/MyObject/partitioning/lonColName", "ra_PS"));
         kv.push_back(make_pair(p + "/MyObject/partitioning/latColName", "decl_PS"));
```

Comment:
I just wanted for this test to continue working after my change to the implementation of `Facade::_containsTable()` method. We could add separate test for different value of table key, I'd prefer to do it when we work on re-designing `Facade` stuff.